documentation/developer-guide/modules/tooling-guide/pages/java-aspect-tooling.adoc

```diff
@@ -439,3 +439,82 @@ characteristicDefinitionsUrl.ifPresent( url -> {
+
+| **Characteristic**  | aas:SubmodelElement, aas:ConceptDescription  | Characteristics in BAMM define the semantics of a property, which includes there types as well as links to further definitions (standards, dictionaries, etc), a natural language description and name in different languages. Type and description are separated in AAS, which is why there is not a one-to-one mapping of a Characteristic to one element in AAS but rather Characteristics are used in the mapping of Properties, first, to guide the generation process and, second, to capture semantics in ConceptDescriptions of properties with data specification "DataSpecificationIEC61360" of the AAS.
+
+| **Collection**  | aas:SubmodelElementCollection, aas:ConceptDescription  | The general remarks to Characteristics apply also for Collection type Characteristics. However, properties referencing Collections are mapped to SubmodelElementCollections. Specific properties of collections are mapped. bamm:Set is unique, bamm:SortedSet is unique and sorted, bamm:List is sorted.
+
+| **Quantifiable**  | aas:SubmodelElement, aas:ConceptDescription | The general remarks to Characteristics apply also for Quantifiable type Characteristics. Quantifiables (also Duration and Measurement) reference a unit, which is added to the ConceptDescription corresponding the mapped Characteristic.
+
+| **Either**  | aas:SubmodelElement, aas:ConceptDescription | The general remarks to Characteristics apply also for Either. However, the Either characteristic has two distinct entries of which one is to be selected. This concept is not present in AAS. Thus both entries will be written to a Submodel template, where one has to be ignored.
+
+| **Trait**  | aas:SubmodelElement, aas:ConceptDescription | The general remarks to Characteristics apply also for Trait. However, the constraint of a trait will be ignored and only the base type will be evaluated, which will act as the characteristic of a property.
+
+| **Code**  | aas:SubmodelElement, aas:ConceptDescription | Similar to plain Characteristic.
+
+| **StructuredValue**  | aas:SubmodelElement, aas:ConceptDescription | The general remarks to Characteristics apply also for StructuredValue. However, AAS has no concpet like deconstruction rule. Thus, the deconstruction rule and the sub properties of the deconstruction entity will be ignored and only the Characteristic is mapped.
+
+| **Enumeration**  | aas:SubmodelElement, aas:ConceptDescription | The general remarks to Characteristics apply also for Enumerations. Additionally, the values of an Enumeration are mapped to a valueList of a DataSpecificationIEC61360.
+
+| **State**  | aas:SubmodelElement, aas:ConceptDescription | Same as Enumeration.
+
+| bamm:MultiLanguageString  | aas:MultiLanguageProperty | if a multilanguage string is used in BAMM it is mapped to the MultiLanguageProperty in AAS.
+|===
+
+
+=== Known Limitations
+The AAS generator implements a base set of features, which are mapped from BAMM to AAS.
```

Comment:
Is it "AAS generator" or "AAS Generator"? In the introductory paragraph, the other spelling was used.